src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
> VSIX should deploy paket.dll to a known place and all dlls that are found over there should be hooked in automatically.

I feel, this will make it a pain for xplat tooling, ionide users would have extra step to perform before this could eventually work, it will bring more user requests because it doesn't work etc.

I'd go for having paket as the first baked-in one, of course the nuget team is welcome to participate and add the handler (that ships by default) as well.